gateway/ln-gateway/src/db.rs

```diff
@@ -508,6 +523,82 @@ async fn migrate_to_v4(mut ctx: MigrationContext<'_>) -> Result<(), anyhow::Erro
     Ok(())
 }
 
+async fn migrate_to_v5(mut ctx: MigrationContext<'_>) -> Result<(), anyhow::Error> {
```

Comment:
Your PR description explains the issue quite well, but it's not documented in the code. Should we add a link to this PR in the documentation for `migrate_to_v5()` so future readers can understand why this migration exists? I know this code is unlikely to substantially change, so the git diff could be used, but I think having the link here would be easier for future readers.

Reply:
I added a comment that should help.